checkout/rust/delivery-customization/default/src/main.rs

```diff
@@ -1,28 +1,35 @@
 use shopify_function::prelude::*;
 use shopify_function::Result;
 
-use graphql_client;
 use serde::{Deserialize, Serialize};
-use serde_json;
 
 generate_types!(
     query_path = "./input.graphql",
     schema_path = "./schema.graphql"
 );
 
 #[derive(Serialize, Deserialize, Default, PartialEq)]
-struct Config {}
+#[serde(rename_all(deserialize = "camelCase"))]
+struct Configuration {
+
+}
+
+impl Configuration {
+    fn from_str(value: &str) -> Self {
+        serde_json::from_str(value).expect("Unable to parse configuration value from metafield")
+    }
+}
```

Comment:
nit: Would it be simpler to not have this as a structure/concept and just operate on the JSON directly? Reduces the indirection a bit for new readers

Reply:
Yeah worth discussing what's most idiomatic in Rust, and try to make that consistent across the Rust templates. For now this is more consistent with Discounts.